server/form-pages/apply/risk-and-need-factors/further-considerations/contingencyPlanQuestions.test.ts

```diff
@@ -1,31 +1,41 @@
 import { ContingencyPlanQuestionsBody } from '../../../../@types/ui'
+import applicationFactory from '../../../../testutils/factories/application'
 import contingencyPlanQuestionsBodyFactory from '../../../../testutils/factories/contingencyPlanQuestionsBody'
 import { itShouldHaveNextValue, itShouldHavePreviousValue } from '../../../shared-examples'
 import ContingencyPlanQuestions from './contingencyPlanQuestions'
 
 describe('ContingencyPlanQuestions', () => {
   const body = contingencyPlanQuestionsBodyFactory.build()
+  const application = applicationFactory.build()
 
   describe('title', () => {
     it('should set the title', () => {
-      const page = new ContingencyPlanQuestions(body)
+      const page = new ContingencyPlanQuestions(body, application)
 
       expect(page.title).toEqual('Contingency plans')
     })
   })
 
   describe('body', () => {
-    const page = new ContingencyPlanQuestions(body)
+    const page = new ContingencyPlanQuestions(body, application)
 
     expect(page.body).toEqual(body)
   })
 
-  itShouldHaveNextValue(new ContingencyPlanQuestions(body), '')
+  describe('if shouldShowTriggerPlanPages returns true', () => {
+    const emergencyApplication = applicationFactory.emergencyApplication().build()
```

Comment:
Technically, as we're testing `shouldShowTriggerPlanPages` elsewhere, we should probably mock this, that way if the implementation changes, we only have to change the tests in one place.